src/app/client/src/app/modules/public/module/offline/services/system-info/system-info.service.ts

```diff
@@ -1,6 +1,6 @@
 import { Injectable } from '@angular/core';
-import { PublicDataService } from '@sunbird/core';
-import { ConfigService } from '@sunbird/shared';
+import { PublicDataService } from '../../../../../core/services/public-data/public-data.service';
```

Comment:
Here also imports are changed

Reply:
There was a circular dependency, due to this, as we pointing to index.ts where all the services were included